examples/src/examples/graphics/area-lights.tsx

```diff
@@ -16,7 +16,7 @@ class AreaLightsExample {
             'normal': new pc.Asset('normal', 'texture', { url: '/static/assets/textures/seaside-rocks01-normal.jpg' }),
             'gloss': new pc.Asset('gloss', 'texture', { url: '/static/assets/textures/seaside-rocks01-gloss.jpg' }),
             'statue': new pc.Asset('statue', 'container', { url: '/static/assets/models/statue.glb' }),
-            'luts': new pc.Asset('luts', 'binary', { url: '/static/assets/binary/area-light-luts.bin' }),
+            'luts': new pc.Asset('luts', 'json', { url: '/static/assets/binary/area-light-luts.json' }),
```

Comment:
If this file is now a `json` file and not a `bin` file, shouldn't it be in a folder called `json` instead of `binary`?

Reply:
right, changing